examples/lib/stories/input/draggables.dart

```diff
@@ -34,6 +34,9 @@ class DraggableSquare extends PositionComponent
 
   @override
   bool onDragUpdate(int pointerId, DragUpdateInfo info) {
+    if (parent is! DraggablesGame) {
```

Comment:
Why is this necessary now?

Reply:
Because I use this game in the game-in-game example, and it becomes very weird when the square is moved to the other game if it can still be dragged around there since the parent is spinning.

Reply:
Oh, this is the example code! Alright